saviSimulation/savi/jason_processing/SAVIWorld_model.java

```diff
@@ -155,26 +155,39 @@ public void setup() {
 		// ======= Jason BDI agents ================
 		Map<String, AgentModel> agentList = new HashMap<String, AgentModel>();
 		//====================================================
-				
+
+
+//		createUGV(agentList, 300, 300);
+		createUAV(agentList, 500, 500);
+		createTree(250, 300);
+		createTree(300, 250);
+		createTree(250, 250);
+		createTree(250, 350);
+//		createTree(350, 250);
+//		createTree(350, 300);
+//		createTree(300, 350);
+//		createTree(350, 350);
```

Comment:
Ignore these, just for debugging.

Reply:
I have exactly those changes on my next PR when i introduce a real scenario.

Reply:
Fully agree with this. Would make debugging the behaviour a lot easier!